### DISCUSSION TITLE
How to prevent scroll to collapse when toggling the collapse via JavaScript?

### DISCUSSION THREAD

**Discussion by lu-x** [twbs/bootstrap#38509] [General]
Version: 5.2.3

I use `new Collapse(this.element, { toggle: true})` to toggle a collapse programatically. Everytime i do the page scrolls to the collapse. Is there a way to prevent this? I already did some research but didn't find a solution so far.

Related docs: https://getbootstrap.com/docs/5.2/components/collapse/#via-javascript

**Comment by gerardnico** [comment]
As far as I understand scrolling, it's created by the browser when a user or javascript clicks on an anchor element with a URL that has an anchor. 
And by preventing the default behavior of the click, it should be enough?
```javascript
event.preventDefault();
```

**Reply by lu-x** [reply]
Hi there,
thanks for your reply. So i'd have to figure out the fired event, add a listener and handle the event properly? Sounds a bit cumbersome to me 🤔
